editors/neovim/Portfile

```diff
@@ -33,20 +33,16 @@ depends_lib             port:gettext \
                         port:libvterm \
                         port:libtermkey \
                         port:unibilium \
-                        port:jemalloc \
-                        port:msgpack1 \
-                        port:lua \
-                        port:lua-lpeg \
-                        port:lua-mpack \
-                        port:lua-luabitop \
-                        port:luv
+                        port:msgpack \
```

Comment:
Also taking the chance to upgrade msgpack. Few ports in the tree are using `msgpack1`.

Reply:
> Also taking the chance to upgrade msgpack. Few ports in the tree are using `msgpack1`.

Since msgpack and msgpack1 conflict, it would have been nice to use the "deactivate hack" or something to prevent upgrade from failing over the conflict